build/media_source/com_scheduler/js/admin-view-run-test-task.es6.js

```diff
@@ -42,7 +42,9 @@ const initRunner = () => {
     const id = parseInt(button.dataset.id, 10);
     const { title } = button.dataset;
 
-    modal.querySelector('.modal-title').innerHTML = Joomla.Text._('COM_SCHEDULER_TEST_RUN_TITLE').replace('%d', id.toString());
+
+    modal.querySelector('.modal-title').innerHTML = Joomla.Text._('COM_SCHEDULER_RUN_TASK_TITLE').replace('%d', id.toString());
+
```

Comment:
This should fix the drone complaining 

```suggestion
    modal.querySelector('.modal-title').innerHTML = Joomla.Text._('COM_SCHEDULER_RUN_TASK_TITLE').replace('%d', id.toString());

```